007_smoothed_Laplace/smoothed_laplace.ipynb

```diff
@@ -162,14 +167,14 @@
     "    laplace_grad_values[i] = (laplace.logpdf(x[i])[0]-laplace.logpdf(x[i]-dx)[0])/dx\n",
     "\n",
     "plt.figure()\n",
-    "plt.subplot(1,2,1)\n",
+    "plt.subplot(2,1,1)\n",
     "plt.plot(x, laplace_pdf_values, '-o', label='Laplace')\n",
     "plt.plot(x, smoothed_laplace_pdf_values, label='Smoothed Laplace')\n",
     "plt.xlabel('x')\n",
     "plt.ylabel('value')\n",
     "plt.title('Log PDF')\n",
     "plt.legend(loc='upper right')\n",
-    "plt.subplot(1,2,2)\n",
+    "plt.subplot(2,1,2)\n",
     "plt.plot(x, laplace_grad_values, '-o', label='Laplace')\n",
     "plt.plot(x, smoothed_laplace_grad_values, label='Smoothed Laplace')\n",
```

Comment:
Please add a brief statement after the value and gradient plots to make the point that we consider that this verifies the implementation, because the values agree and the gradient values also agree, noting that around zero the gradient values differ slightly with the smoothed version begin continuous in the gradient unlike Laplace which has a discontinuity.

Reply:
Done